packages/kit/src/core/create_app/index.js

```diff
@@ -35,21 +39,30 @@ export function create_app({ manifest_data, output, cwd = process.cwd() }) {
 }
 
 /**
+ * Returns string without leading tabs and whitespace.
  * @param {string} str
+ *
  */
 function trim(str) {
 	return str.replace(/^\t\t/gm, '').trim();
 }
 
 /**
- * @param {ManifestData} manifest_data
- * @param {string} base
+ * Generates code for the `manifest.js` file which imports svelte components and exports the app's `routes`, an an array of arrays containing a pattern to match the route and two arrays of components (`[__layout, page]` and `[error]`).
+ * @param {ManifestData} manifest_data - object containing paths to the apps assets, user defined components, routes, and the app's __layout and error components.
+ * @param {string} base - the base path to prepend to imports
```

Comment:
it doesn't import anything. we could simplify these descriptions a lot
```suggestion
 * Generates code for the `manifest.js` file which exports the app's `routes` and the root layout/error components.
 * @param {ManifestData} manifest_data - a listing of paths to the app's components and assets.
 * @param {string} base - the base path to prepend to imports
 * @return {string}
```

Reply:
My bad  - I glanced at the array and saw the import statements, didn't see they were wrapped in functions.